packages/pyright-internal/src/analyzer/program.ts

```diff
@@ -1947,10 +1850,15 @@ export class Program {
                 return editActions;
             }
 
+            const referenceProvider = new ReferencesProvider(this, token);
             const renameMode = this._getRenameSymbolMode(sourceFileInfo, referencesResult, isDefaultWorkspace);
             switch (renameMode) {
                 case 'singleFileMode':
-                    sourceFileInfo.sourceFile.addReferences(referencesResult, true, this._evaluator!, token);
+                    referenceProvider.addReferencesToResult(
+                        sourceFileInfo.sourceFile.getFilePath(),
+                        /*includeDeclaration*/ true,
```

Comment:
Eric wants these with space. /* includeDeclaration */

Reply:
thank you